wgpu-hal/src/gles/queue.rs

```diff
@@ -94,6 +91,14 @@ impl super::Queue {
                         view.mip_levels.start as i32,
                         view.array_layers.start as i32,
                     );
+                } else if target == glow::TEXTURE_CUBE_MAP {
+                    gl.framebuffer_texture_2d(
```

Comment:
nit: could be merged more closely with the other `framebuffer_texture_2d`

Reply:
i figured it was clearer this way than doing `if target == glow::TEXTURE_CUBE_MAP { texture = CUBE_MAP_FACES[...]; }` but feel free to make your own judgment